pages/Banner.js

```diff
@@ -43,6 +43,23 @@ export default class Banner {
 		return this;
 	}
 
+	clickUseOfFundsLink() {
+		this.dandy.click( bannerConfig.selectors.use_of_funds.link );
+		return this;
+	}
+
+	checkUseOfFundsIsVisible() {
+		this.dandy.checkElementExists( bannerConfig.selectors.use_of_funds.modal_is_open );
```

Comment:
All you need to check here is if the modal is open as that's what this test is for. You can leave the selectors in the config though, as we might write a test that needs to select them in the future.